src/PDOWrapper.php

```diff
@@ -915,6 +915,11 @@ public function getErrorMessage() {
 	public function getPDOException() {
 		return $this->pdo_exception;
 	}
+
+	public function error($message){
+		//error_log($message);
+		echo $message;
```

Comment:
Generally we don't want to write errors to `php://output` [(which is where echo writes)](http://www.php.net/manual/en/wrappers.php.php) by default because in most PHP configurations (e.g. with Apache and mod_php), this information will end up being written to the web page itself. That's not good for production environments. However, we could add a static member, PRINT_ERRORS that when set to true echos the error messages rather than writing to the error log. But in this case PRINT_ERRORS should default to false.